src/main/scopes/wc/utils/get-wc-prefix.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { wcPrefixMap } from '../wc-prefix-map.js'
+
+/**
+ * Return the expected web component prefix value based on a given JS import path
+ *
+ * @param path - The JS import path (string)
+ * @returns - The expected web component tag prefix (string) or ''
+ */
+export function getWcPrefix(path: string): string {
+  const importPathPatterns = [...wcPrefixMap.keys()]
+  const match = importPathPatterns.find((key: RegExp) => key.test(path))
+  let prefix
+  if (match) {
+    prefix = wcPrefixMap.get(match)
+  }
+  return prefix ?? ''
+}
```

Comment:
```suggestion
export function getWcPrefix(path: string): string {
  for (const [pattern, prefix] of wcPrefixMap) {
    if (pattern.test(path)) {
      return prefix
    }
  }
  return ''
}
```

This can be cleaned up a little bit, the old way works fine, but it was doing some extra work by spreading the keys into an array and then doing a second lookup with .get(), which isn’t super efficient. Since Map preserves order, we can just loop through the entries directly and return the prefix as soon as we find a match. We wanna save every little bit of performance just in case!